crates/autopilot/src/infra/database/auction/mod.rs

```diff
@@ -0,0 +1,24 @@
+use {
+    self::postgres::dto,
+    super::Database,
+    crate::domain::{self},
+};
+
+pub mod postgres;
```

Comment:
I think the module structure should be `infra::persistence::postgres::dto::{auction, order, fee, quote}`